components/filter/filter-overflow-group.js

```diff
@@ -0,0 +1,26 @@
+import { css, LitElement } from 'lit';
+import { OverflowGroupMixin } from '../overflow-group/overflow-group-mixin.js';
+
+/**
+ * A component that can be used to display a group of filters that will be put into an overflow filter when they no longer fit on the first line of their container
+ * @slot - d2l-filters to be added to the container
```

Comment:
What happens if they try to put something other than `d2l-filter` in the `slot`?  From the next PR it looks like we won't add it to the overflow group, but it'll be rendered until it overflows and then disappear?  It's an invalid state so maybe this is "good enough" to tell the consumer they messed up, as long as no one uses it as feature

Reply:
Yup that's correct as to how it currently behaves. We could also add a `console.warn` in `convertToOverflowItem` about how this item is invalid here and will just disappear once space is no longer available for it.